accessibility.md

```diff
@@ -4,24 +4,24 @@ permalink: /accessibility/
 title: Accessibility Statement
 ---
 
-*You may also be interested in the [**venue accessibility statement**](http://www.focusterra.ethz.ch/en/your-visit.html).*
-
 RustFest is an inclusive conference and as such strives to be accessible to everyone. This includes:
 
 * Offering significantly discounted tickets for concession holders.
-* Ensuring our venues are accessible by wheelchair.
-* Making sure that maps and orientation material is available.
-* Providing clear signage to, from and through the building.
+* Ensuring all our venues are accessible by wheelchair.
+* Live transcription of all talks.
+    - Appropriate measures can be taken for the workshop day and the impl days. Please get in touch.
+* Makeing sure that maps and orientation material is available.
+* Providing clear signage to, from and through the building
 * Providing quiet spaces for those with sensory issues.
 * Providing accessibility for conference sessions.
-* Providing complimentary tickets for assistants for disabled participants.
+* Providing complimentary tickets for assistants for those with disabilities.
+* Access to non-gendered toilets.
```

Comment:
At this point, can we already guarantuee that for the main venue?

Reply:
I'm not sure, I'll rephrase

Reply:
https://github.com/RustFestEU/rome.rustfest.eu/pull/2/commits/89379ce871c15c2b20bec27c87fc7aa59817394d